build-js.sh

```diff
@@ -73,876 +72,49 @@ OUT="$PWD/out"
 
 echo "generate embedded intrinsics module"
 
-python3 $BINARYEN_SCRIPTS/embedwast.py $BINARYEN_SRC/passes/wasm-intrinsics.wast $BINARYEN_SRC/passes/WasmIntrinsics.cpp
+python $BINARYEN_SCRIPTS/embedwast.py $BINARYEN_SRC/passes/wasm-intrinsics.wast $BINARYEN_SRC/passes/WasmIntrinsics.cpp
```

Comment:
This change seems unrelated.

Reply:
Ah, right, needed to change this so it runs within my Debian VM since there's no python3 apparently. Do you want me to change it back?

Reply:
yes please.  If we are going to change python versions we should do it deliberately in its own PR.